subprojects/language-native/src/main/java/org/gradle/language/rc/tasks/WindowsResourceCompile.java

```diff
@@ -201,15 +202,17 @@ public void setMacros(Map<String, String> macros) {
     }
 
     /**
-     * Additional arguments to provide to the compiler.
+     * <em>Additional</em> arguments to provide to the compiler.
+     *
+     * @since 5.1
      */
     @Input
-    public List<String> getCompilerArgs() {
+    public ListProperty<String> getCompilerArgs() {
         return compilerArgs;
     }
 
     public void setCompilerArgs(List<String> compilerArgs) {
```

Comment:
It's OK to remove this method.  We just need to change `WindowsResourcesCompileTaskConfig` from
```
task.setCompilerArgs(rcCompiler.getArgs());
```
to
```
task.getCompilerArgs().set(rcCompiler.getArgs());
```

Reply:
Ok, I removed the setter per your instructions.  That is interesting chaining the setter to the getter.